packages/plugins/sentry/src/index.ts

```diff
@@ -92,6 +104,12 @@ export const useSentry = (options: SentryPluginOptions = {}): Plugin => {
   const renameTransaction = pick('renameTransaction', false);
   const skipOperation = pick('skip', () => false);
   const skipError = pick('skipError', defaultSkipError);
+  const cachePrintedDocuments = pick('cachePrintedDocuments', false);
```

Comment:
I think caching the document from `onParse` might be better.

Reply:
What do you mean? This MR is about caching the "expensive" `print()`, not any parsing

Reply:
```ts
const documentSDLMap = new WeakMap<DocumentNode, string>();
{
  onParse({ params }) {
      return ({ result }) => {
         documentSDLMap.set(result, params.source.toString());
      }
  },
  onExecute({ args }) {
    const sdl = documentSDLMap.get(args.document);
  }
}
```

Reply:
Sadly this is not quite the same, as the result from `print()` is formatted whereas this could be any raw minified input sdl.

Reply:
Persisted operations set the `DocumentNode` so the same object is used then it works exactly like you said if I don't miss anything.

Reply:
Yeah, but my point is that you can't use `params.source.toString()` since that's most likely a non-indented blob input, so there needs to be at least one call to `print()`, which might as well be this one.